ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/BlockResultFactory.java

```diff
@@ -173,6 +173,7 @@ public EngineGetPayloadResultV4 payloadTransactionCompleteV4(
         txs,
         blockWithReceipts.getBlock().getBody().getWithdrawals(),
         blockWithReceipts.getBlock().getBody().getDeposits(),
+        blockWithReceipts.getBlock().getBody().getExits(),
```

Comment:
Looking to this class it seems that both `depositRoot` and `exitsRoot` are missing in `BlockResult`

Reply:
Both `deposits_root` and `exits_root` are part of the BlockHeader, not BlockBody.

Reply:
If we want to add those to the BlockResult, we should consider opening a PR to update https://github.com/ethereum/execution-apis/blob/main/src/schemas/block.yaml